packages/regenesis-surgery/package.json

```diff
@@ -21,6 +21,7 @@
     "@discoveryjs/json-ext": "^0.5.3",
     "@eth-optimism/contracts": "^0.4.10",
     "@uniswap/v3-core-optimism": "^1.0.0-rc.0",
+    "axios": "^0.22.0",
```

Comment:
`axios` has been deprecated for a while, I would use `node-fetch` if possible.

Reply:
will swtich!